src/app/models/threshold-indicator-query-opts.model.ts

```diff
@@ -1,9 +1,16 @@
+import { ClimateModel } from './climate-model.model';
 import { IndicatorQueryOpts } from './indicator-query-opts.model';
 
 export interface ThresholdIndicatorQueryOpts extends IndicatorQueryOpts {
     params: {
         threshold: Number;
         threshold_units: string;
         threshold_comparator: string;
+
+        // from base
```

Comment:
I originally did this as well after the TS2.4 upgrade, but then went further down the rabbit hole and refactored IndicatorQueryOpts such that IndicatorQueryOpts.params is now `interface IndicatorParams` instead of `any`, and can be extended by `ThresholdIndicatorParams`. `ThresholdIndicatorQueryOpts` disappears. 

No action necessary here, just more of a heads up that a larger change related to this is coming in the PR for #212. It enforces type checking far better in a few places, and avoids the awkwardness of having to redefine parameters as is done here.